package.json

```diff
@@ -55,7 +55,7 @@
     "test:ci:vitest:unit": "vitest run --coverage tests/unit/",
     "test:ci:vitest:integration": "vitest run --coverage tests/integration/",
     "e2e": "node ./tools/e2e/run.js",
-    "docs": "node ./site/scripts/docs.mjs",
+    "docs": "pushd site && npm run build && popd",
```

Comment:
This is weird because this repo isn't quite a monorepo but the site is treated as a pseudo-monorepo-package. I prefer this over reaching into scripts 🤷🏼.